doc/policies_list/3.3.0/policies.json

```diff
@@ -0,0 +1,1276 @@
+        "configuration": {
+          "definitions": {
+            "condition": {
+              "$id": "#/definitions/condition",
+              "description": "condition to be evaluated",
+              "properties": {
+                "combine_op": {
+                  "default": "and",
+                  "enum": [
+                    "and",
+                    "or"
+                  ],
+                  "type": "string"
+                },
+                "operations": {
+                  "items": {
+                    "$ref": "#/definitions/operation"
+                  },
+                  "minItems": 1,
+                  "type": "array"
+                }
+              },
+              "type": "object"
+            },
+            "operation": {
+              "$id": "#/definitions/operation",
+              "properties": {
+                "left": {
+                  "type": "string"
+                },
+                "left_type": {
+                  "default": "plain",
+                  "description": "How to evaluate 'left'",
+                  "oneOf": [
+                    {
+                      "enum": [
+                        "plain"
+                      ],
+                      "title": "Evaluate 'left' as plain text."
+                    },
+                    {
+                      "enum": [
+                        "liquid"
+                      ],
+                      "title": "Evaluate 'left' as liquid."
+                    }
+                  ],
+                  "type": "string"
+                },
+                "op": {
+                  "enum": [
+                    "==",
+                    "!="
+                  ],
+                  "type": "string"
+                },
+                "right": {
+                  "type": "string"
+                },
+                "right_type": {
+                  "default": "plain",
+                  "description": "How to evaluate 'right'",
+                  "oneOf": [
+                    {
+                      "enum": [
+                        "plain"
+                      ],
+                      "title": "Evaluate 'right' as plain text."
+                    },
+                    {
+                      "enum": [
+                        "liquid"
+                      ],
+                      "title": "Evaluate 'right' as liquid."
+                    }
+                  ],
+                  "type": "string"
+                }
+              },
+              "required": [
+                "left",
+                "op",
+                "right"
+              ],
+              "type": "object"
+            }
+          },
+          "properties": {
+            "condition": {
+              "$ref": "#/definitions/condition"
+            },
+            "policy_chain": {
+              "description": "the policy chain to execute when the condition is true",
```

Comment:
We need to have a more consistent capitalization in the UI. Can you please review descriptions like this to ensure they begin with a capital letter?
_The policy chain to execute when the condition is true_